docker/nginx/nginx.conf

```diff
@@ -1,68 +1,51 @@
-user  nginx;
-worker_processes  1;
-
-error_log  /var/log/nginx/error.log warn;
-pid        /var/run/nginx.pid;
+user nginx;
+worker_processes auto;
+error_log /var/log/nginx/error.log;
+pid /run/nginx.pid;
 
 events {
-    worker_connections  1024;
+    worker_connections 1024;
 }
 
 http {
-    include       /etc/nginx/mime.types;
-    default_type  application/octet-stream;
-
     log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
-                      '$status $body_bytes_sent "$http_referer" '
-                      '"$http_user_agent" "$http_x_forwarded_for"';
+        '$status $body_bytes_sent "$http_referer" '
+        '"$http_user_agent" "$http_x_forwarded_for"';
+
+    log_format trace '$remote_addr - $time_iso8601 "$request" '
+        '$status  $body_bytes_sent "$http_referer"  '
+        '"$http_user_agent" "$http_x_forwarded_for" rid=$request_id '
+        'rt=$request_time uct="$upstream_connect_time" '
+        'uht="$upstream_header_time" urt="$upstream_response_time" '
+        'sid=$upstream_http_x_session_id uid=$upstream_http_x_user_id';
 
-    access_log  /var/log/nginx/access.log  main;
+    access_log  /var/log/nginx/access.log  trace;
 
-    sendfile        on;
-    tcp_nopush      on;
-    tcp_nodelay     on;
+    sendfile            on;
+    tcp_nopush          on;
+    tcp_nodelay         on;
+    keepalive_timeout   65;
+    types_hash_max_size 2048;
 
-    keepalive_timeout  65;
+    include             /etc/nginx/mime.types;
+    default_type        application/octet-stream;
 
     gzip on;
     gzip_disable "msie6";
     gzip_http_version 1.1;
-    gzip_comp_level 5; # or anything between 4-6
```

Comment:
Is this comment misleading?

Reply:
No indeed removing it was not intended, I just added it back